openfisca_us/variables/finance/benefit/ccdf.py

```diff
@@ -0,0 +1,130 @@
+from openfisca_core.model_api import *
+from openfisca_us.entities import *
+from openfisca_us.tools.general import *
+from openfisca_us.variables.demographic.spm_unit import *
+
+
+class ccdf_asset_eligible(Variable):
+    value_type = bool
+    entity = SPMUnit
+    definition_period = YEAR
+    documentation = ""
+
+    def formula(spm_unit, period, parameters):
+        # Get state and poverty ratio for SPM unit.
+        state = spm_unit("spm_unit_state", period)
+        assets = spm_unit("spm_unit_assets", period)
+        # Get parameters.
+        p_asset_limit = parameters(period).benefit.CCDF.eligibility.assets[
```

Comment:
```suggestion
        p_asset_limit = parameters(period).benefit.ccdf.eligibility.assets[
```
Once #229 is merged, all variables and parameters will be lowercase, so you'll need to update a few places here.